app/views/layouts/_nav.html.erb

```diff
@@ -0,0 +1,3 @@
+<% Category.all.each do |category| %>
+  <%= link_to category.name, root_path %>
```

Comment:
Is this a placeholder for now?

Reply:
Yes. We dont have a show for categories yet.

Reply:
outside of scope